docs/plugins/ingest-geoip.asciidoc

```diff
@@ -203,3 +203,18 @@ Which returns:
 }
 --------------------------------------------------
 // TESTRESPONSE
+
+[[ingest-geoip-settings]]
+===== Settings
+
+The geoip processor supports the following settings:
+
+`ingest.geoip.cache_enabled`::
+
+    Whether to enable caching of results. Defaults to `false`.
```

Comment:
Ok, then let's align ES here with the behavior in Logstash. I'll enable it by default then (as per your suggestion with a single setting).